### ISSUE TITLE
omni does not abort renewslice when wrong password is entered multiple times

### ISSUE THREAD

**Issue by tcmitchell** [GENI-NSF/geni-tools#43]
A user reported that omni did not abort after she had failed to type in her private key password 3 times. When a user types in their password incorrectly 3 times, the entire process should probably be aborted.

The following log demonstrates the problem:
```
$ ./src/omni.py renewslice my_test 20111130T18:00:00 -c ~/.gcf/omni_config_plc
INFO:omni:Loading config file /home/user/.gcf/omni_config_plc
INFO:omni:Using control framework plc
INFO:omni.sfa:SFA Registry: http://www.planet-lab.org:12345
INFO:omni.sfa:SFA Slice Manager: http://www.planet-lab.org:12347
Enter PEM pass phrase:
ERROR:omni.sfa:Wrong pass phrase for private key.
INFO:omni.sfa:.... please retry.
Enter PEM pass phrase:
ERROR:omni.sfa:Wrong pass phrase for private key.
ERROR:omni.sfa:Wrong pass phrase after 2 tries. Cannot do Get SFA user credential from registry http://www.planet-lab.org:12345 for user plc.bbn.user using cert file /home/user/.gcf/plc.bbn.user.gid.
ERROR:omni.sfa:Cannot get a slice credential without a user credential
ERROR:omni.sfa:Failed to renew slice urn:publicid:IDN+plc:bbn+slice+my_test: could not get a slice credential
WARNING:omni:Failed to renew slice my_test
Enter PEM pass phrase:
Enter PEM pass phrase:
INFO:omni:Slice urn:publicid:IDN+plc:bbn+slice+my_test expires on 2011-11-30 17:00:00 UTC
INFO:omni: ------------------------------------------------------------
INFO:omni: Completed renewslice:

Options as run:
      configfile: /home/user/.gcf/omni_config_plc
      framework: plc
      native: True

Args: renewslice my_test 20111130T18:00:00

Result Summary: Failed to renew slice my_test
Slice urn:publicid:IDN+plc:bbn+slice+my_test expires on 2011-11-30 17:00:00 UTC
INFO:omni: ============================================================
```

 _Imported from trac ticket [#43](http://trac.gpolab.bbn.com/gcf/ticket/43),  created by tmitchel on 06-17-2011 at 10:04, last modified: 07-06-2011 at 09:07_

**Comment by tcmitchell** [comment]
Fixed in [bb6469f1597be9511e2d5b76dc15c2a4905ddf69](https://github.com/GENI-NSF/geni-tools/commit/bb6469f1597be9511e2d5b76dc15c2a4905ddf69)

_Trac comment by tmitchel (github user: tcmitchell) on 06-17-2011 at 10:15_

**Comment by luisanevers** [comment]
Verified in gcf.1.3.1

_Trac comment by lnevers (github user: luisanevers) on 07-05-2011 at 17:25_
